management/sites/index.php

```diff
@@ -1,5 +1,5 @@
 <?php
-$root = realpath($_SERVER['DOCUMENT_ROOT']);
+$root = realpath($_SERVER["DOCUMENT_ROOT"]);
```

Comment:
I think we should prefer using single quotes for constant strings, and reserve double quotes for string-interpolated strings.